cdap-ui/app/tracker/tracker.html

```diff
@@ -44,22 +44,12 @@
     <loading-icon></loading-icon>
     <loading-indicator></loading-indicator>
     <status-alert-message></status-alert-message>
+    <global-footer></global-footer>
```

Comment:
needs to do iframe check

Reply:
Lines 44 to 47 are already in a `<div ng-if="!$state.params.iframe">`